content/en/docs/concepts/networking/function-entrypoints.md

```diff
@@ -57,32 +44,39 @@ curl -sv -HHOST:$FUNC_HOST http://$IP$FUNC_PATH
 
 ### Access functions by the external address
 
-#### Configure DNS
-You have two options to configure DNS:
-
-- Option 1: Real DNS
-
-    The real DNS configuration is based on the network topology of your environment.
-
-- Option 2: Configure the local hosts file
-
-    Get Function's HOSTNAME and Gateway's ip:
-    ```shell
-    export HOSTNAME="function-sample.default.ofn.io"
-    export IP=$(kubectl get node -l node-role.kubernetes.io/worker= -o=jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}')
-    ```
-    
-    Append this line to /etc/hosts:
-    ```shell
-    echo $IP  $HOSTNAME | sudo tee -a /etc/hosts
-    ```
-
-#### Access Function
-Get `Function` external address by running following command:
+First, you should choose and configure a DNS based on your network environment:
```

Comment:
To access a sync function by the external address, you'll need to configure DNS first. Either Magic DNS or real DNS works: